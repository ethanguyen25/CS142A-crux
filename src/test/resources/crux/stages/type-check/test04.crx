// test add

void main() {
  printInt(1 + 1);
}
