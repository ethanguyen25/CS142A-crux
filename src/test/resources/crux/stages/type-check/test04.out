Crux Program has no type errors.
ast.DeclarationList(3)
  ast.FunctionDefinition(3)[Symbol(main:func(TypeList()):void), []]
    ast.StatementList(4)
      void: ast.Call(4)[Symbol(printInt)]
        int: ast.OpExpr(4)[ADD]
          int: ast.LiteralInt(4)[1]
          int: ast.LiteralInt(4)[1]
